enterprise/views.py

```diff
@@ -774,6 +782,9 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             organization_logo = None
             organization_name = None
 
+        if enterprise_course_enrollment and not enterprise_course_enrollment.consent_granted:
+            add_consent_declined_message(request, enterprise_customer, course_details)
```

Comment:
@douglashall shouldn't we show consent decline message only when a learner declines the consent? instead of showing consent decline message always if the learner has not provided course level consent.
There could be the case that the learner has provided account level consent.